public/app/features/dashboard/components/PanelEditor/VisualizationSelectPane.tsx

```diff
@@ -103,6 +123,17 @@ export const VisualizationSelectPane = ({ panel, data }: Props) => {
                 onClose={() => {}}
               />
             )}
+            {listMode === VisualizationSelectPaneTab.Widgets && isWidgetEnabled && (
```

Comment:
I found a future issue here where, if the user has in the localStorage the widget listMode after enabling the feature flag, if the removes that feature flag he won't see any VizTypePicker.